src/work-allocation-2/containers/task-home/task-home.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRouteSnapshot, NavigationEnd, Router, RoutesRecognized } from '@angular/router';
+import { SubNavigation } from '@hmcts/rpx-xui-common-lib';
+import { Observable, Subscription } from 'rxjs';
+import { share } from 'rxjs/operators';
+
+import { AppUtils } from '../../../app/app-utils';
+import { TaskSortField } from '../../models/tasks';
+
+@Component({
+  selector: 'exui-task-home',
+  templateUrl: 'task-home.component.html'
+})
+export class TaskHomeComponent implements OnInit, OnDestroy {
+
+  /**
+   * Take in the Router so we can navigate when actions are clicked and
+   * to identify which sub-navigation item to highlight.
+   */
+  private readonly MY_TASKS: SubNavigation = { text: 'My tasks', href: '/mywork/list', active: true };
+  /**
+   * The sub-navigation items.
+   */
+  public subNavigationItems: SubNavigation[] = [
+    this.MY_TASKS,
+    { text: 'Available tasks', href: '/mywork/available', active: false }
```

Comment:
_Nitpick:_
`/mywork/available` doesn't look like an appropriate path for stuff that is _not_ work currently assigned to _me_. 🤔